frontend/src/features/leases/detail/LeasePages/details/DetailTermInformationBox.tsx

```diff
@@ -48,7 +47,7 @@ const StyledDate = styled.div`
 
 const StyledTermInformationBox = styled.div`
   width: 25.5rem;
-  background-color: ${props => props.theme.css.primaryColor};
+  background: rgba(0, 33, 66, 0.73);
```

Comment:
This is okay for now but would love to see it added to the **colors.scss** file so we can re-use it when needed

Reply:
This is just the primary color with an alpha filter applied, I'm not convinced that deserves its own entry.